node/src/chain_spec.rs

```diff
@@ -812,14 +989,33 @@ fn testnet_genesis(
         parachain_info: datahighway_parachain_runtime::ParachainInfoConfig {
             parachain_id: id,
         },
-        aura: AuraConfig { authorities: initial_authorities },
+        collator_selection: CollatorSelectionConfig {
+            invulnerables: invulnerables.iter().cloned().map(|(acc, _)| acc).collect(),
+            candidacy_bond: EXISTENTIAL_DEPOSIT * 16,
```

Comment:
how was this candidacy bond determined?

Reply:
I am not sure. I have asked here https://github.com/paritytech/subport/issues/301

Reply:
why is this resolved when the issue is still open?

Reply:
I have created issue https://github.com/DataHighway-DHX/node/issues/248 for tracking